tests/test_assets.py

```diff
@@ -1376,6 +1450,538 @@ def test_error_message_plurality(self,
+        num_exchanges = 3
+        metadata = pd.DataFrame.from_records(
+            [
+                # first sid per country
+                {
+                    'symbol': 'A',
+                    'asset_name': 'Asset A',
+                    'start_date': T('2014-01-01'),
+                    'end_date': T('2014-01-05'),
+                },
+                {
+                    'symbol': 'B',
+                    'asset_name': 'Asset B',
+                    'start_date': T('2014-01-06'),
+                    'end_date': T('2014-01-10'),
+                },
+
+                # second sid per country
+                {
+                    'symbol': 'C',
+                    'asset_name': 'Asset C',
+                    'start_date': T('2014-01-01'),
+                    'end_date': T('2014-01-05'),
+                },
+                {
+                    'symbol': 'A',  # claiming the unused symbol 'A'
+                    'asset_name': 'Asset A',
+                    'start_date': T('2014-01-06'),
+                    'end_date': T('2014-01-10'),
+                },
+            ] * num_exchanges,
+            index=np.repeat(np.arange(num_exchanges * 2), 2),
+        )
+        metadata['exchange'] = np.repeat(
+            ['EXCHANGE %d' % n for n in range(num_exchanges)],
+            4,
+        )
+        exchanges = pd.DataFrame({
+            'exchange': ['EXCHANGE %d' % n for n in range(num_exchanges)],
+            'country_code': [
+                self.country_code(n) for n in range(num_exchanges)
+            ]
+        })
+        self.write_assets(equities=metadata, exchanges=exchanges)
+        finder = self.asset_finder
+
+        # note: these assertions walk forward in time, starting at assertions
+        # about ownership before the start_date and ending with assertions
+        # after the end_date; new assertions should be inserted in the correct
+        # locations
+
+        # no one held 'A' before 01
+        with self.assertRaises(SymbolNotFound):
+            finder.lookup_symbol('A', T('2013-12-31'))
+        for n in range(num_exchanges):
+            with self.assertRaises(SymbolNotFound):
+                finder.lookup_symbol(
+                    'A',
+                    T('2013-12-31'),
+                    country_code=self.country_code(n),
+                )
+
+        # no one held 'C' before 01
+        with self.assertRaises(SymbolNotFound):
+            finder.lookup_symbol('C', T('2013-12-31'))
+        for n in range(num_exchanges):
+            with self.assertRaises(SymbolNotFound):
+                finder.lookup_symbol(
+                    'C',
+                    T('2013-12-31'),
+                    country_code=self.country_code(n),
+                )
+
+        for asof in pd.date_range('2014-01-01', '2014-01-05', tz='utc'):
+            # from 01 through 05 the first sid on the exchange held 'A'
+            with self.assertRaises(MultipleSymbolsFound):
```

Comment:
Why does this raise for multiple symbols if the first sid on the exchange held A?

Reply:
Ah, I see, this is a conflict between exchanges. I wonder if it might be helpful (both here and generally for a consumer) to make a subclass of `MultipleSymbolsFound` that's specific to the case of "multiple symbols found in different countries". Making it a subclass means that calling code can still catch `MultipleSymbolsFound` and have it do the right thing, but it might be easier to track down an error and/or figure out a fix if you know that the conflict is cross-market rather than cross-time.